src/test/java/org/synyx/urlaubsverwaltung/ui/SickNoteIT.java

```diff
@@ -0,0 +1,272 @@
+import static org.testcontainers.containers.BrowserWebDriverContainer.VncRecordingMode.RECORD_FAILING;
+
+@Testcontainers
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@ContextConfiguration(initializers = UITestInitializer.class)
+class SickNoteIT {
+
+    @LocalServerPort
+    private int port;
+
+    @Container
+    private final BrowserWebDriverContainer<?> browserContainer = new BrowserWebDriverContainer<>()
+        .withRecordingMode(RECORD_FAILING, new File("target"))
+        .withCapabilities(new FirefoxOptions());
+
+    static final TestMariaDBContainer mariaDB = new TestMariaDBContainer();
+
+    @DynamicPropertySource
+    static void mariaDBProperties(DynamicPropertyRegistry registry) {
+        mariaDB.start();
+        mariaDB.configureSpringDataSource(registry);
+    }
+
+    @Autowired
+    private PersonService personService;
+    @Autowired
+    private AccountInteractionService accountInteractionService;
+    @Autowired
+    private WorkingTimeWriteService workingTimeWriteService;
+    @Autowired
+    private MessageSource messageSource;
+
+    @Test
+    void ensureSickNote() {
+        final Person person = createPerson();
+
+        final RemoteWebDriver webDriver = browserContainer.getWebDriver();
+        final WebDriverWait wait = new WebDriverWait(webDriver, 20);
+
+        final LoginPage loginPage = new LoginPage(webDriver, messageSource, ENGLISH);
+        final NavigationPage navigationPage = new NavigationPage(webDriver);
+
+        webDriver.get("http://host.testcontainers.internal:" + port);
+
+        wait.until(pageIsVisible(loginPage));
+        loginPage.login(new LoginPage.Credentials(person.getUsername(), "secret"));
+
+        wait.until(pageIsVisible(navigationPage));
+        assertThat(navigationPage.quickAdd.hasPopup()).isTrue();
+
+        sickNote(webDriver, person);
+        sickNoteWithIncapacityCertificate(webDriver, person);
+        childSickNote(webDriver, person);
+        childSickNoteWithIncapacityCertificate(webDriver, person);
+
+        sickNoteStatisticListView(webDriver, person);
+
+        navigationPage.logout();
+        wait.until(pageIsVisible(loginPage));
+    }
+
+    private void sickNote(RemoteWebDriver webDriver, Person person) {
+        final WebDriverWait wait = new WebDriverWait(webDriver, 20);
+
+        final NavigationPage navigationPage = new NavigationPage(webDriver);
+        final SickNotePage sickNotePage = new SickNotePage(webDriver);
+        final SickNoteDetailPage sickNoteDetailPage = new SickNoteDetailPage(webDriver, messageSource, ENGLISH);
+
+        navigationPage.quickAdd.click();
+        navigationPage.quickAdd.newSickNote();
+        wait.until(pageIsVisible(sickNotePage));
+
+        assertThat(sickNotePage.personSelected(person.getNiceName())).isTrue();
+        assertThat(sickNotePage.typeSickNoteSelected()).isTrue();
+        assertThat(sickNotePage.dayTypeFullSelected()).isTrue();
+
+        final int currentYear = LocalDate.now().getYear();
```

Comment:
clock?

Reply:
ich würde das nun nicht überall hin schreiben, da müsste man dann noch drüber und schauen wo überall. Es sollte aber `LocalDate.now(clock).getYear()` sein und die clock sollte in UTC sein, bzw autowired